packages/project-integrity/src/fiori-project/index.ts

```diff
@@ -0,0 +1,118 @@
+import { existsSync } from 'fs';
+import { join } from 'path';
+import { getCapCustomPaths } from '@sap-ux/project-access';
+import {
+    checkProjectIntegrity,
+    disableProjectIntegrity,
+    enableProjectIntegrity,
+    initProject,
+    isProjectIntegrityEnabled,
+    updateProjectIntegrity
+} from '../integrity';
+import type { CheckIntegrityResult, Content } from '../types';
+
+export const fioriIntegrityDataPath = join('.fiori-ai/ai-integrity.json');
+
+/**
+ * Get the list of files to protect the integrity of.
+ *
+ * @param projectRoot - root folder of the project
+ * @returns - list of file paths
+ */
+async function getFileList(projectRoot: string): Promise<string[]> {
+    const fileList: string[] = [];
+
+    const schemaCds = join(projectRoot, 'db/schema.cds');
+    if (existsSync(schemaCds)) {
+        fileList.push(schemaCds);
+    } else {
+        throw new Error(`File ${schemaCds} does not exist.`);
+    }
+    const servicesCds = join(projectRoot, 'srv/service.cds');
+    if (existsSync(servicesCds)) {
+        fileList.push(servicesCds);
+    } else {
+        throw new Error(`File ${servicesCds} does not exist.`);
+    }
+
+    return fileList;
+}
+
+/**
+ * Returns additional string content, like the CAP environment.
+ * This content will be stored in the integrity data.
+ *
+ * @param projectRoot - root folder of the project
+ * @returns - additional content to store in the integrity data
+ */
+async function getAdditionalStringContent(projectRoot: string): Promise<Content> {
+    const capCustomPaths = await getCapCustomPaths(projectRoot);
+    return { capPaths: JSON.stringify(capCustomPaths) };
+}
+
+/**
+ * Initialize a Fiori project for integrity protection.
+ *
+ * @param projectRoot - root folder of the project
+ */
+export async function initFioriProject(projectRoot: string): Promise<void> {
+    const integrityFilePath = join(projectRoot, fioriIntegrityDataPath);
```

Comment:
probably should not hardcode `fioriIntegrityDataPath`. Use it as default instead.

Reply:
Thanks for reviewing @devinea!
The module has low level functionality where you can freely define the list of files and the location to store integrity data, see [packages/project-integrity/src/integrity/index.ts](https://github.com/SAP/open-ux-tools/pull/2630/files#diff-85861fb8d93603583b2b7a3e3b336f8743173f185aca598eb7c12435218ddc87 ).
Additionally, it has convenience functions for Fiori project generated with Fiori tools AI, in which the list of files and path to integrity data is fixed, see [packages/project-integrity/src/fiori-project/index.ts](https://github.com/SAP/open-ux-tools/pull/2630/files#diff-0a17122e243ee9ce9f5c9907f2f0700f6d779efec1490d6fec419a3d2cc921a3).